MELATONIN_VBLANK cause issues on windows due to lamda issues in perfetto

```
C:...\modules\melatonin_inspector/melatonin/components/fps_meter.h(58): warning C4458: declaration of 'kCatIndex_ADD_TO_PERFETTO_DEFINE_CATEGORIES_IF_FAILS_58' hides class member
C:\...\modules\melatonin_inspector/melatonin/components/fps_meter.h(58): note: see declaration of 'melatonin::FPSMeter::visibilityChanged::<lambda_1>::()::<lambda_1>::kCatIndex_ADD_TO_PERFETTO_DEFINE_CATEGORIES_IF_FAILS_58'
C:\...\modules\melatonin_inspector/melatonin/components/fps_meter.h(58): warning C4189: 'kCatIndex_ADD_TO_PERFETTO_DEFINE_CATEGORIES_IF_FAILS_58': local variable is initialized but not referenced
C:\...\modules\melatonin_inspector/melatonin/components/fps_meter.h(60): error C2065: 'kCatIndex_ADD_TO_PERFETTO_DEFINE_CATEGORIES_IF_FAILS_58': undeclared identifier
```

If `MELATONIN_VBLANK`  did a check before assigning to 1 I could set this to 0 to avoid the error.  Once Perfetto fix the lambda issue this will work on windows.  Once the dependency is updated of course.

--- a/melatonin/components/fps_meter.h
+++ b/melatonin/components/fps_meter.h
@@ -4,7 +4,9 @@
 
 // VBlank was added in 7.0.3
 #if (JUCE_MAJOR_VERSION >= 7) && (JUCE_MINOR_VERSION >= 1 || JUCE_BUILDNUMBER >= 3)
+    #ifndef MELATONIN_VBLANK
     #define MELATONIN_VBLANK 1
+    #endif
 #else
     #define MELATONIN_VBLANK 0
 #endif

